### PR TITLE
Create search notes #22

### PR DESCRIPTION
**Pull request by SimonLab**
ref #22 

Create/update/edit notes for search terms

### CODE REVIEW

File: web/controllers/note_search_controller.ex [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    case Repo.insert(changeset) do
+      {:ok, _note} ->
+        conn
+        |> redirect(to: history_path(conn, :index))
+      {:error, changeset} ->
+        search = Repo.get!(Search, note_params["search_id"])
+        render conn, "new.html", changeset: changeset, search: search
+    end
+    conn
+    |> redirect(to: history_path(conn, :index))
```
Comment (shouston3) [inline web/controllers/note_search_controller.ex]: Are we reaching this line? I would have thought we will end on either line 17 or 20?

Comment (SimonLab) [inline web/controllers/note_search_controller.ex]: Thanks for spotting that, I must have missed this when rebasing

File: web/router.ex [hunk 2]
```diff
@@ -32,6 +32,16 @@ defmodule Bep.Router do
...
+  scope "/note", Bep do
+    pipe_through [:browser, :authenticate_user]
+    resources "/search", NoteSearchController
+  end
+
+  scope "/note", Bep do
+    pipe_through [:browser, :authenticate_user]
+    resources "/search", NoteSearchController
+  end
+
```
Comment (shouston3) [inline web/router.ex]: Don't think we need this twice

**Comment by shouston3** [comment]
Reassign when addressed 👍

**Pull request merged**